tests/test_features/test_pca.py

```diff
@@ -122,14 +122,13 @@ def test_transform_without_fit(self):
     @pytest.mark.xfail(IS_WINDOWS_OR_CONDA, reason="RMS of 12.115 on miniconda")
     def test_pca_decomposition_quick_method(self):
         """
-        Test the quick method PCA visualizer 2 dimensions scaled.
+        Test the quick method PCA visualizer
```

Comment:
I am curious as to why you decided to change this test. The previous test still works with the quick method changes that you made.

The new test that you added is failing because of the following:
1) the load_credit dataset needs to be added to the import list
2) the baseline image test used for the image comparison test is based on the old test so it is completely different from the image that is generated by the new test you added.

If you would like to replace the old test with the new one let me know and I can give you instructions on how to update the baseline image so that your new test passes.  Otherwise, please revert back to the old test.